mkdocs.yml

```diff
@@ -1,13 +1,34 @@
 site_name: Redbox
 site_description: Bringing Generative AI to the way the Civil Service works
 site_author: Incubator for AI
-repo_url: https://github.com/i-dot-ai/redbox-copilot
+repo_url: https://github.com/i-dot-ai/redbox
 
 theme:
+  
+  logo: assets/briefcase_white.svg
+
+
   name: "material"
+  font:
+    text: "Inter"
+    code: "DM Mono"
   palette:
-    primary: "red"
-    accent: "red"
+    # Palette toggle for light mode
+    - media: "(prefers-color-scheme: light)"
+      scheme: default
+      primary: custom
+      toggle:
+        icon: material/weather-sunny
+        name: Switch to dark mode
+
+    # Palette toggle for dark mode
+    - media: "(prefers-color-scheme: dark)"
```

Comment:
Can we lighten the red used for links in Dark Mode? It's not super visible against the background. I think #FF3838 should pass.

Reply:
Great shout. Have added it in!